index.html

```diff
@@ -4,10 +4,12 @@
   <meta charset='utf-8' />
   <title>Roadmap</title>
   <link rel='stylesheet' href='./components/Table/style.css'>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
Would be better to add external styles before internal to have an ability to override them